frictionless/schema/field.py

```diff
@@ -257,6 +257,8 @@ def metadata_validate(cls, descriptor):
             type = descriptor.get("type")
             Class = system.select_field_class(type)
             field = Class(name=descriptor["name"])
```

Comment:
Can we use `Class(name=descriptor.get("name"), format=descriptor.get('format'))`?

Note that `format` is not required property so it might cause a runtime error is we use `descriptor["name"]`